core/src/main/scala/pureconfig/package.scala

```diff
@@ -143,9 +166,7 @@ package object pureconfig {
    */
   @throws[ConfigReaderException[_]]
   def loadConfigOrThrow[Config](path: Path)(implicit conv: ConfigConvert[Config], ct: ClassTag[Config]): Config = {
```

Comment:
I can't figure out where the `ClassTag` is getting used. `loadConfig` doesn't require it. I think it can go.

I think that's also true of all (or most) of the other `loadConfigOrThrow` methods.